componentsCss/slider.css

```diff
@@ -0,0 +1,41 @@
+@import '../style.css';
+
+.slidecontainer {
+    width: 100%;
+    margin: 10px;
+    padding: 10px;
+  }
+  
+  .slider {
+    -webkit-appearance: none;
+    width: 80%;
+    height: 15px;
+    border-radius: 5px;
+    background: var(--darkgray-color);
+    outline: none;
```

Comment:
using px values , any specific reason for not using rems?

Reply:
below 15px, i think its better to use px